docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

```diff
@@ -33,6 +33,17 @@ Modeling Strategy for the Vitamin A Deficiency Cause
 Cause Hierarchy
 ^^^^^^^^^^^^^^^
 
+.. image:: vitA_cause_hierarchy.svg
```

Comment:
To be consistent, its better to use Vitamin A deficiency in the diagram.

Reply:
thanks for the suggestion